src/js/components/account/reset-password.js

```diff
@@ -0,0 +1,98 @@
+import { API } from "../../api";
+import {
+  getAnchorElement,
+  getDiv,
+  getForm,
+  getInput,
+  getLabel,
+  getSubmitButton,
+} from "../../utils/element-factory";
+
+export class ResetPassword {
+  constructor() {
+    const $successTemplate = $(".styles .form-styles .w-form-done").clone();
+    const $failTemplate = $(".styles .form-styles .w-form-fail").clone();
+    const $loginAnchor = getAnchorElement(
+      "/accounts/login/",
+      "link-text form-submit",
+      "Please log in with your new password"
+    );
+    $loginAnchor.hide();
+
+    const defaultBaseUrl = "https://muni-portal-backend.openup.org.za";
```

Comment:
we should be using the base url in the API - can you let the API take care of this?

Reply:
ohhh this is here because we're letting the form submit rather than submitting using ajax.

I think expose a method in the API to give you the base URL rather than hardcoding it here

Reply:
I know this is the case with the other forms - if it's very easy, please update them to use this new exposed backend URL.

Reply:
@jbothma I've done some refactoring in #92 that addresses this, it adds a `buildBaseApiUrl` function incl. promptapi support.

We can do some git-foo to get both of these in at the same time, but what I would suggest is that we merge this pr and #92 and then in a separate ticket we do the refactoring next week for the forms to use the new function. This would keep things simpler for me and keeps the diffs cleaner.

Is that fine with you?

Reply:
You can see it here https://github.com/OpenUpSA/muni-portal-frontend/blob/d31b2cb9842c3a72355f077016aaa1e9d7c1af1f/src/js/api.js